pyproject.toml

```diff
@@ -1,3 +1,62 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[tool.flit.module]
+name = "aiida_cp2k"
+
+[project]
+name = "aiida-cp2k"
+dynamic = ["version", "description"]
+readme = "README.md"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Framework :: AiiDA",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: POSIX :: Linux",
+    "Operating System :: MacOS :: MacOS X",
+    "Programming Language :: Python :: 3",
+]
```

Comment:
Probably also good to specify minimum required version of Python
```
requires-python = '>=3.8'
```

Reply:
done.